packages/cardano-services/test/Asset/TypeOrmNftMetadataService.test.ts

```diff
@@ -0,0 +1,32 @@
+import { TypeOrmNftMetadataService, createDnsResolver, getConnectionConfig, getEntities } from '../../src';
+import { logger, mockProviders } from '@cardano-sdk/util-dev';
+
+describe('TypeOrmNftMetadataService', () => {
+  let service: TypeOrmNftMetadataService;
+
+  beforeAll(async () => {
+    const dnsResolver = createDnsResolver({ factor: 1.1, maxRetryTime: 1000 }, logger);
+    const entities = getEntities(['nftMetadata', 'asset', 'block']);
```

Comment:
```suggestion
    const entities = getEntities(['asset']);
```

This **should** be enough as the other two tables are dependencies of the `asset` one. With _should_ I mean I'm not 100% sure, this needs to be tested.

Reply:
Works with only `asset`. Fixed on 670b25a